src/io/files.js

```diff
@@ -740,6 +740,56 @@ p5.prototype.httpPost = function () {
  * @param  {function}      [errorCallback] function to be executed if
  *                                    there is an error, response is passed
  *                                    in as first argument
+ *
+ *
+ * @example
+ * <div>
+ * <code>
+ * // Examples use USGS Earthquake API:
+ * // https://earthquake.usgs.gov/fdsnws/event/1/#methods
+ *
+ * // displays an animation of all USGS earthquakes
+ * var earthquakes;
+ * var eqFeatureIndex = 0;
+ *
+ * function preload() {
+ *    var url = 'https://earthquake.usgs.gov/fdsnws/event/1/query?format=geojson';
+ *    httpDo(url,
+ *      {
+ *        method: 'GET',  // http method - "GET", "PUT", "POST", etc.
```

Comment:
let's omit this comment for readability, since this info is mentioned in the parameters already

Reply:
Just the method comment or the Request options one as well?

Reply:
I think the other one is fine to leave, since those options may be less clear